### PR TITLE
Reenabled 'OrientGraphStructureStandardTest' and fixed 'EdgeTest'

### PR DESCRIPTION
**Pull request by velo**
I took the falling tests and did my best to get them working.

Lemme know if this is an acceptable merge.

Also, do you have a standard code formatter for this project?

### CODE REVIEW

File: driver/src/main/java/org/apache/tinkerpop/gremlin/orientdb/OrientEdgeIterator.java [hunk 1]
```diff
@@ -88,7 +88,7 @@ public OrientEdge createGraphElement(final Object iObject) {
                 edge = new OrientEdge(this.sourceVertex.getGraph(), rec.getIdentity(), (OIdentifiable) this.sourceVertex.id(),
                     connection.getValue());
         } else if (immutableSchema.isEdgeType()) {
-            edge = new OrientEdge(this.sourceVertex.getGraph(), rec.getIdentity(), connection.getValue());
+            edge = new OrientEdge(this.sourceVertex.getGraph(), rec.getRecord(), connection.getValue());
```
Comment (mpollmeier) [inline driver/src/main/java/org/apache/tinkerpop/gremlin/orientdb/OrientEdgeIterator.java]: was this change really necessary? loading the record will take longer than just the ID - the idea is to only load the record if necessary.

File: driver/pom.xml [hunk 2]
```diff
@@ -68,6 +68,17 @@
             <version>4.12</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```
Comment (mpollmeier) [inline driver/pom.xml]: these dependencies don't seem to be used?

**Comment by mpollmeier** [comment]
It's good to reenable the test to see where we are at and get a direction for going forward. I wouldn't merge this to master though because there are still 33 test failures.

**Comment by mpollmeier** [comment]
I don't have a standard code formatter for this, and as I'm not really a java guy any more, I'm not sure what the best option for this would be. Do you have any suggestions?

**Comment by mpollmeier** [comment]
here's the test failures as I get them: 
[org.apache.tinkerpop.gremlin.orientdb.OrientGraphStructureStandardTest.txt](https://github.com/mpollmeier/orientdb-gremlin/files/22873/org.apache.tinkerpop.gremlin.orientdb.OrientGraphStructureStandardTest.txt)

File: driver/pom.xml (hunk 2 above)
Comment (velo) [inline driver/pom.xml]: I had to change the version, cause I was getting some methodNotFoundExceptions since mockito-all embeds hamcrest

File: driver/src/main/java/org/apache/tinkerpop/gremlin/orientdb/OrientEdgeIterator.java (hunk 1 above)
Comment (velo) [inline driver/src/main/java/org/apache/tinkerpop/gremlin/orientdb/OrientEdgeIterator.java]: @mpollmeier 
how does it feel now?

**Comment by velo** [comment]
@mpollmeier  yes, I'm working on fixing the failures and errors.
May be, we should merge to another branch... or we can merge it broken and work to get it less broken.

**Comment by velo** [comment]
@mpollmeier what about https://github.com/revelc/formatter-maven-plugin ?

File: driver/pom.xml (hunk 2 above)
Comment (mpollmeier) [inline driver/pom.xml]: cool, that makes sense

**Comment by mpollmeier** [comment]
I'd like to have master green, but in the best shape possible. how about you just disable the test again and we merge your changes in? then you continue working on your branch to fix the other tests. once all tests are green we enable the test in master.

**Comment by velo** [comment]
works for me

**Comment by mpollmeier** [comment]
ok, just update this PR then and I'll merge it in

**Comment by velo** [comment]
@mpollmeier 
I got a greenish build....

The tests still run, but right now, it ignore that one failure.  How does that sound?

Tests run: 776, Failures: 31, Errors: 238, Skipped: 243

**Comment by mpollmeier** [comment]
sounds like a good middle ground :)
